src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const empoyees = [...document.querySelectorAll('li')];
```

Comment:
The variable name 'empoyees' seems to be a typo. It should be 'employees'. Correct variable naming is crucial for code readability and maintenance.